examples/jsm/renderers/common/Renderer.js

```diff
@@ -176,6 +182,18 @@ class Renderer {
 
 		//
 
+		const coordinateSystem = this.coordinateSystem;
+
+		if ( camera.coordinateSystem !== coordinateSystem ) {
+
+			camera.coordinateSystem = coordinateSystem;
+
+			camera.updateProjectionMatrix();
```

Comment:
@mrdoob I fixed  ( https://github.com/mrdoob/three.js/issues/23372#issuecomment-1556075016 ) issue here. If the camera has updated the `projectionMatrix` using the `WebGL` coordinate system, it will now update again with the `WebGPU` coordinate system.